src/components/GHListLogo/styled.ts

```diff
@@ -24,3 +24,103 @@ export const LogoImage = styled.img`
         font-family: 'Mabry Pro';
     }
 `;
+
+export const MapGuideButton = styled('div')`
+    margin-left: 15px;
+    margin-top: 7px;
+    .help-guide-button {
```

Comment:
There is no need to add those styles under custom class name, all styles can be added to the `MapGuideButton`